src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -555,6 +580,46 @@ PyChipError pychip_DeviceController_SetCheckMatchingFabric(bool check)
     return ToPyChipError(CHIP_NO_ERROR);
 }
 
+PyChipError pychip_DeviceController_SetIcdRegistrationParameters(chip::Controller::DeviceCommissioner * devCtrl, bool enabled,
+                                                                 uint8_t * icdSymmetricKeyOrNull, uint64_t icdCheckInNodeIdOrZero,
```

Comment:
- icdSymmetricKeyOrNull --> icdTokenOrNull
- token has to be `const uint8_t *`
- Please have a size_t for token size, rather than assume size, for safety.

Reply:
updated.